README.md

```diff
@@ -164,3 +164,21 @@ Please read [the full documentation](https://github.com/openeuropa/openeuropa) f
 ## Versioning
 
 We use [SemVer](http://semver.org/) for versioning. For the available versions, see the [tags on this repository](https://github.com/openeuropa/oe_bootstrap_theme/tags).
+
+## Upgrade from older versions
+
+### Upgrade from 1.0.0-alpha7 and earlier
+
+Target release: You can upgrade directly to the version that contains this README.md.
```

Comment:
What does this mean? If there's nothing to do, then the line doesn't need to exist.